CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add python code for tensorflow DNNEstimator
 - Ability to run a subflow as if it were an operation using the
   `dffml.dataflow.run` operation.
+- Doctestable examples for an Abstract Base Class in `Base Source`.
```

Comment:
```suggestion
- Doctestable examples for `BaseSource`.
```

Reply:
We probably don't need to call out that it's an Abstract Base Class, also there's no space